h2o-py/h2o/estimators/aggregator.py

```diff
@@ -7,6 +7,7 @@
 from __future__ import absolute_import, division, print_function, unicode_literals
 
 from h2o.estimators.estimator_base import H2OEstimator
+from h2o.estimators.targetencoder import H2OTargetEncoderEstimator
```

Comment:
@sebhrusen for hook to be working we need this import as well. But we don't need import for TargetEncoder estimator for example and, theoretically, for some other models (but for now I'm trying to make all estimators to accept te_model_id parameter). Will try to exclude te in gen_python script. But if we go away from validation against type, there will be no need for this at all

Reply:
@sebhrusen yeah, excluding by algo name is not a problem.